src/video_core/renderer_opengl/texture_filters/texture_filterer.cpp

```diff
@@ -0,0 +1,79 @@
+/// Copyright 2020 Citra Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <functional>
+#include "common/logging/log.h"
+#include "video_core/renderer_opengl/texture_filters/anime4k/anime4k_ultrafast.h"
+#include "video_core/renderer_opengl/texture_filters/bicubic/bicubic.h"
+#include "video_core/renderer_opengl/texture_filters/texture_filterer.h"
+#include "video_core/renderer_opengl/texture_filters/xbrz/xbrz_freescale.h"
+
+namespace OpenGL {
+
+using TextureFilterContructor = std::function<std::unique_ptr<TextureFilterBase>(u16)>;
+
+template <typename T>
+std::pair<std::string_view, TextureFilterContructor> FilterMapPair() {
```

Comment:
```suggestion
static std::pair<std::string_view, TextureFilterContructor> FilterMapPair() {
```

or enclose all internal details with an anonymous namespace to give this internal linkage.